components/elm/cime_config/config_component.xml

```diff
@@ -83,12 +83,10 @@
       <value compset="SSP370(?:SOI)?_EAM%CMIP6_ELM">2015-2100_SSP370_transient</value>
       <value compset="2010(?:SOI)?_EAM%CMIP6-LR_ELM">2010_CMIP6LR_control</value>
       <value compset="2010(?:SOI)?_EAM%CMIP6-HR_ELM">2010_CMIP6HR_control</value>
-      <value compset="2010(?:SOI)?_EAM.*SCREAM-LR" >2010_defscream_control</value>
-      <value compset="2010(?:SOI)?_EAM.*SCREAM-HR" >2010_defscream_control</value>
-      <value compset="2010(?:SOI)?_EAM.*SCREAM-LR-DYAMOND2" >2010_dy2scream_control</value>
-      <value compset="2010(?:SOI)?_EAM.*SCREAM-HR-DYAMOND2" >2010_dy2scream_control</value>
-      <value compset="2000(?:SOI)?_EAM.*SCREAM-LR" >2000_control</value>
-      <value compset="2000(?:SOI)?_EAM.*SCREAM-HR" >2000_control</value>
+      <!-- Use cases for SCREAM compsets -->
+      <value compset="2010(?:SOI)?_.*SCREAM"                    >2010_defscream_control</value>
+      <value compset="2010(?:SOI)?_EAM.*SCREAM-[L,H]R-DYAMOND2" >2010_dy2scream_control</value>
```

Comment:
I'm not familiar with the `[L,H]` syntax to point to both L or H options. In L23 above, it looks like there's no comma (`[LH]`). Can you verify this is what we want/tested?

Reply:
Ah good catch, I think what I wrote here matches "L", ",", or "H". I'll fix and retest.

Reply:
Okay, fixed.